blueprints/data-solutions/mongodb-cluster/init/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM google/cloud-sdk:slim
```

Comment:
Is it too much of a bother to do this without a dedicated image, installing the needed stuff at runtime? It would avoid the need of building images to run the blueprint.

Reply:
I think it might be feasible with a lot of work (passing all the files via cloud-init or GCS), but the container-based solution is much cleaner (and faster) and we wouldn't be pushing the maximum length of metadata value.